### PR TITLE
added sortBy hypeScore

### PR DESCRIPTION
**Pull request by haritonasty**
image

### CODE REVIEW

File: src/components/Trends/GetHypedTrends.js [hunk 1]
```diff
@@ -2,18 +2,34 @@ import React from 'react'
...
 
+const sortByHype = items => {
+  const sortedData = items[0].topWords.sort(sort('score'))
+  return {
+    ...items,
+    topWords: sortedData
+  }
+}
+
+const sort = sortBy => (a, b) => simpleSort(a[sortBy], b[sortBy])
```
Comment (Partysun) [inline src/components/Trends/GetHypedTrends.js]: Move this to util file. @haritonasty

Comment (Partysun) [reply src/components/Trends/GetHypedTrends.js]: And find one more place in our code, where we have this function and replace it with util method.

File: src/components/Trends/GetHypedTrends.js [hunk 2]
```diff
@@ -2,18 +2,34 @@ import React from 'react'
...
 
 class GetHypedTrends extends React.Component {
   render () {
-    const { render, ...rest } = this.props
-    return render(rest)
+    const { render, items, ...rest } = this.props
+    const trends = items.slice(-1)
+    const props = {
+      ...rest,
+      items: trends.length > 0 ? [sortByHype(trends)] : trends
```
Comment (Partysun) [inline src/components/Trends/GetHypedTrends.js]: I don't understand this case. If trends.length <= 0, trends === []
why did you write ? ... : trends ?

Also you return only one array from items. What were items before we returned?
Can we create text for this.

Comment (haritonasty) [reply src/components/Trends/GetHypedTrends.js]: lat question: https://github.com/santiment/app/blob/feature/hype-score-sort/src/components/Trends/TrendsTable/TrendsTables.js

this component expect array ("trends" prop), but it really not nesessary, as I understand. Should I rewrite this logic here?
inside array: datetime, topWords, __typename fileds

File: src/components/Trends/GetHypedTrends.js [hunk 3]
```diff
@@ -2,18 +2,32 @@ import React from 'react'
...
```
Comment (DmitriVanGuard) [inline src/components/Trends/GetHypedTrends.js]: This line breaks the logic of multiple trend snapshots. If `trends.length > 0 ?` is true, then `items` will always be a one item array

Comment (haritonasty) [reply src/components/Trends/GetHypedTrends.js]: oh, yes, now I understand for what was 3 arrays

**Pull request merged**